tanjun/injecting.py

```diff
@@ -240,11 +240,11 @@ def __init__(self, callback: CallbackSig[_T], /) -> None:
         self._is_async: typing.Optional[bool] = None
         self._descriptors = self._parse_descriptors(callback)
 
-    # This is delegated to the callback in-order to delegate set/list behaviour for this class to the callback.
+    # This is delegated to the callback to delegate set/list behaviour for this class to the callback.
```

Comment:
I noticed this comment seemed to be confusing (in-order can mean sequentially), and `in order to` is just redundant words.

(I could probably make it better but that's way out of scope lol)